demos/projects/ST/b-l475e-iot01a/sample_gsg_device.c

```diff
@@ -36,7 +36,7 @@
 #define TELEMETRY_GYROSCOPEY        ( "gyroscopeY" )
 #define TELEMETRY_GYROSCOPEZ        ( "gyroscopeZ" )
 
-const char * pcModelId = "dtmi:azureiot:devkit:freertos:gsgstml475;1";
+const char * pcModelId = "dtmi:azureiot:devkit:freertos:gsgstml475;2";
 
 const char * pcManufacturerPropertyValue = "STMicroelectronics";
```

Comment:
>pcManufacturerPropertyValue [](http://example.com/codeflow?start=13&length=27)

should we keep strings as const char or macros

Reply:
Currently they are stored as strings, as they linker is used to link in the correct version. We could move to using the preprocessor, would possibly require additional #ifdefs and such which would look a little messier